modules/openy_features/openy_location/modules/openy_loc_camp/openy_loc_camp.install

```diff
@@ -24,6 +24,34 @@ function openy_loc_camp_update_dependencies() {
   return $dependencies;
 }
 
+/**
+ * Implements hook_install().
+*/
+function openy_loc_camp_install() {
+  $config_factory = Drupal::configFactory();
+  if (\Drupal::service('module_handler')->moduleExists('openy_map')) {
+    // Add camp as active type to display camps on location finder map.
+    $config_factory
+      ->getEditable('openy_map.settings')
+      ->set('active_types.camp', 'Camps')
+      ->save(TRUE);
+  }
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function openy_loc_camp_uninstall() {
+  $config_factory = Drupal::configFactory();
+  if (\Drupal::service('module_handler')->moduleExists('openy_map')) {
+    // Remove camp form active type on map.
+    $config_factory
```

Comment:
For this better to use $config_updater service, otherwise config will be mark as manual edited